### ISSUE TITLE
Question: More indicators/annotations/shapes

### ISSUE THREAD

**Issue by varnikaj** [d3fc/d3fc#1690]
Hi,

We would like to contribute more technical indicators/annotations/plots/shapes to the project. Are you open to adding them in the project?

**Comment by ColinEberhardt** [comment]
Hi @varnikaj - absolutely, we are very happy to receive contributions. If we can help at all, please do get in touch.

**Comment by varnikaj** [comment]
Also, do you guys have a plan to move to typescript soon?

**Comment by chrisprice** [comment]
No. There is an outstanding issue for TypeScript definitions but no plan to move over (#590). There are a few d3/d3fc patterns that are problematic to represent in TypeScript (e.g. rebind).
